form.yml.erb

```diff
@@ -135,52 +197,82 @@ attributes:
       - [
           "4.0.2", "app_rstudio_server/4.0.2",
           data-option-for-owens: true,
-          data-option-for-pitzer: true
+          data-option-for-pitzer: true,
+          data-option-for-kubernetes: true,,
```

Comment:
I think these extra commas are throwing the yaml parsing off?

Reply:
I fixed, was likely existing trailing comma and I was copy+paste the items and must have missed.